app/assets/stylesheets/_screen.scss

```diff
@@ -623,3 +624,42 @@ div#keycut-help {
     margin-bottom: 5px;
   }
 }
+
+.project-stories {
```

Comment:
I think that move it to a `_project-stories.scss` file would be good. Things on this file are too messed up and we should actually take components out of here to their own file.